### ISSUE TITLE
Cannot get podman-machine to get proxy settings

### ISSUE THREAD

**Issue by tardich** [containers/podman-desktop#5676]
### Bug description

Using Podman-Desktop 1.7.0, I'm trying to deploy a Podman Machine. I'm behind a corporate proxy. I have two problems:

1. I can't get the podman machine image to download automatically
2. If I manually download an image manually, I can get the podman machine to run, but the proxy settings (/etc/profile.d/default-env.sh) shows as:
    export http_proxy="http://http://my.corporate.proxy:8083"

My proxy settings, in Settings:Proxy are set as an URL, as requested by the tool, such as:

- Web Proxy (HTTP): http://my.corporate.proxy:8083
- Secure Web Proxy (HTTPS): http://my.corporate.proxy:8083

### Operating system

Windows 10

### Installation Method

Installer from website/GitHub releases

### Version

1.7.0

### Steps to reproduce

1) Install Podman Desktop from the installer
2) Configure Proxy settings
3.1) Try to install the Podman Machine
3.2) Manually download a proxy image
3.3) Install the Podman Machine from local image
4) Run the Pod Machine
5) Connect to the Podman machine (wsl -d podman-machine-default) and validate the proxy settings (cat /etc/profile.d/default-env.sh)

**Comment by benoitf** [comment]
@tardich should you use the airgap installer ?

**Comment by tardich** [comment]
Maybe, but how would that change the fact that it cannot:

1) Install the podman machine by automatically download an image
2) Run the podman machine with the proxy settings set correctly in this podman machine

**Comment by jeffmaury** [comment]
Issue seems related to https://github.com/containers/podman-desktop/blob/8f29800fb18e2adf6445dba1e3ee703d0b6b2d6e/packages/main/src/plugin/util/exec.ts#L56

**Comment by tardich** [comment]
In the meantime, I'm running this from Powershell:

```
podman machine stop
$env:http_proxy="10.0.21.80:3213"
$env:https_proxy="10.0.21.80:3213"
$env:no_proxy="127.0.0.1,192.168.0.0/16"
podman machine start
```

to restart podman-machine with the correct proxy parameters. 

That fixed problem #2. For problem #1, I have no clue, so downloaded the image manually and started it from a local image then.
